impl/src/main/java/io/kroxylicious/testing/kafka/testcontainers/TestcontainersKafkaCluster.java

```diff
@@ -216,7 +216,7 @@ private KafkaContainer buildKafkaContainer(KafkaClusterConfig.ConfigHolder holde
                 // KAFKA_LOG_DIR overrides a key in the quarkus kafka image application.properties. The quarkus app uses
                 // that to set log.dir. Any value we set for log.dir in server.properties is lost.
                 .withEnv("KAFKA_LOG_DIR", getBrokerLogDirectory(holder.getBrokerNum()))
-                .withEnv("QUARKUS_LOG_LEVEL", "DEBUG") // Enables org.apache.kafka logging too
+//                .withEnv("QUARKUS_LOG_LEVEL", "DEBUG") // Enables org.apache.kafka logging too
```

Comment:
Revert or remove?

Reply:
Just to make sure as I think there may have been a push while the review was in flight. Commenting this out aligns in with `main` is that what you were looking for or were you asking for the debug logging to be re-enabled?

Reply:
Yeah, I think this one is stale. I was just wanting to avoid having commented out code.

Reply:
Ok will. remove the commented out line from here and the ZK container

Reply:
fixed with https://github.com/kroxylicious/kroxylicious-junit5-extension/pull/167/commits/8be2a6cf2ac5b17ed3d10598019f30cfa573fb96